### DISCUSSION TITLE
SOC alerts not generating

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#5585] [Unsupported Versions]
I am not sure what has changed but the alerts I use to see are no longer generating. Now the most active alert is FILE ZIP Detected with only 1700 events in the past 24 hours. We are using Suricata. Any idea what might have changed as to why the alerts are no longer generating? the FILE ZIP is showing coming from the suricata module and i see traffic passing the interfaces. 

![image](https://user-images.githubusercontent.com/18444612/134041420-db48b603-6654-4cd0-abc1-24ed74a8191a.png)

**Comment by clairmont32** [comment]
Whats the output of `so-status`?

**Reply by wtl06639** [reply]
so-status is showing all green on both manager and sensors

**Reply by clairmont32** [reply]
Ok, and going into Kibana and searching beyond 24h to say -3d or going into Grid shows there's been no traffic logging drops or health issues from the sensors?

**Reply by wtl06639** [reply]
yes. same thing. It appears to have been like this for several weeks. nothing stands out in the logs on manager or sensors. Not sure if some how any config changed that results in the alerts not showing the same? Any conf files you can think of i should check for that

**Comment by dougburks** [comment]
It's possible that your rule update failed and that Suricata is running with no rules enabled. Try running `sudo so-rule-update` manually.
